qt/aqt/webview.py

```diff
@@ -750,7 +750,8 @@ def cleanup(self) -> None:
         except ImportError:
             # this will fail when __del__ is called during app shutdown
             return
-
+        if mw.col:
+            mw.col.set_wants_abort()
```

Comment:
This is the simplest solution I could think of. @dae do you see any problems here?

Reply:
It's a bit surprising to have that called for every webview, and may come back to bite us in the future. Can we catch the close event on the dialogs that display the relevant webviews instead?